cli/cmd/install.go

```diff
@@ -125,10 +125,16 @@ var p platform
 var installCmd = &cobra.Command{
 	Use:   "install",
 	Short: "Installs Keptn on a Kubernetes cluster",
-	Long: `Installs Keptn on a Kubernetes cluster
+	Long: `The Keptn CLI allows installing Keptn on Azure Kubernetes Services (AKS), Amazon Elastic Kubernetes Service (EKS), Google Kubernetes Engine (GKE), Pivotal Container Service (PKS), and on OpenShift.
```

Comment:
Please add any Kubernetes derivate to which your kube config is pointing to.

Reply:
fixed.